libmscore/stafftype.cpp

```diff
@@ -240,6 +242,7 @@ void StaffType::write(XmlWriter& xml) const
             xml.tag("stemsDown",        _stemsDown);
             xml.tag("stemsThrough",     _stemsThrough);
             xml.tag("upsideDown",       _upsideDown);
+            xml.tag("showTabFingering",       _showTabFingering);
```

Comment:
Add an `if (_showTabFingering)` here and the default case won't get written. I believe this only worked without that here in master as some mtests, nameyly tst_guitarpro, are disabled?